### ISSUE TITLE
Has bace dataset of moleculenet been changed in the past  year?

### ISSUE THREAD

**Issue by zhenyuhe00** [deepchem/deepchem#2917]
I'm conducting experiments on bace dataset of moleculenet. I use the same code written one year ago but get much different results. So I wonder if it's because the bace dataset (the smiles of it) has been changed in the past year.

Thank you in advance!

**Comment by hzy-hzy** [comment]
> The SMILES have not changed, but models trained will depend on the underlying numpy/pytorch/tensorflow versions which can change dramatically over a year. There is also randomness due to choice of random seed which can induce variability

Thank you for your quick reply! Do you mean the split of train/valid/test has been changed?

**Comment by rbharath** [comment]
If you use a random splitter, the split will be different each time the code is run.  The dataset itself has not changed
